### PR TITLE
[TransferLearning/doc] Update documentation

### PR DESCRIPTION
**Pull request by kparichay**
Update documentation for the transfer learning application

**Self evaluation:**
1. Build test: [x]Passed [ ]Failed [ ]Skipped
2. Run test: [x]Passed [ ]Failed [ ]Skipped

Signed-off-by: Parichay Kapoor <pk.kapoor@samsung.com>

### CODE REVIEW

File: Applications/TransferLearning/Draw_Classification/README.md [hunk 1]
```diff
@@ -1,51 +1,65 @@
 # Classification of hand-drawn images
 
-Unlike using kNN for the classifier at #55 , we could add new layer at the end of feature extractor.
-In this toy example, fully connected layer is added to classify as below. The size of the feature is also same with previous example and the training set and test set are also same. ( 3 classes, 5 training set and 2 test set of each )
-Only fully connected layers are updatable and Mobilenet ssd v2 is used for feature extractor like previous example and all the testing and training is done on the Galaxy S8.
+Draw Classification example performs classification on the hand-drawn images and maps them to different emotions.
+The application uses a Mobilenet-V2 model pre-trained on ms-coco dataset.
+Transfer learning to used to take advantage of the pre-training model to learn the new information of the
```
Comment (zhoonit) [inline Applications/TransferLearning/Draw_Classification/README.md:5]: Transfer learning is used to?

File: Applications/TransferLearning/Draw_Classification/README.md [hunk 2]
```diff
@@ -1,51 +1,65 @@
...
+hand-drawn image dataset efficiently.
 
-The activation is softmax. Mean Squared Error loss function and Stochastic Gradient Decent is used for the loss function and optimizer. Two fully connected layers are added for the hidden layer and iterate 300 times to make is more simple.
+## Application Details
+
+Below are the steps to perform transfer learning in this application:
+
+1. The pre-trained model is converted from pb format (Tensorflow trained model file) to TFLite format (Tensorflow-Lite frozen model). The last layer of the model is removed in this conversion to allow transfer learning to be done.
+2. The training data is passed through the tflite model and the outputs cached. As the transfer learning only trains the last few layers, the outputs from the non-trainable layers are cached to save computation. This is achieved using the single-shot C-API of NNStreamer for TIZEN. In case of non-tizen OS, Tensorflow-Lite framework c++ API are directly used.
```
Comment (zhoonit) [inline Applications/TransferLearning/Draw_Classification/README.md:13]: the outputs cached -> the outputs are cached?

File: Applications/TransferLearning/Draw_Classification/README.md [hunk 3]
```diff
@@ -1,51 +1,65 @@
...
-### Neural Network Configuration
 The configuration of the example is below,
 <p align = "center">
 <img src="https://github.com/nnstreamer/nntrainer/blob/master/docs/images/02a7ee80-f0ce-11e9-97b8-bcc19b7eb222.png" width="400" height="250" > </p>
 
 ### Resource Data
 
-Training and test data for application is in ```Applications/TransferLearning/Draw_Classification/res`.
-Model configuration for the applicaiton is ```Applications/TransferLearning/Draw_Classification/res/Training.ini`.
+Training and test data for application is located in ```Applications/TransferLearning/Draw_Classification/res`.
```
Comment (zhoonit) [inline Applications/TransferLearning/Draw_Classification/README.md:32]:
```suggestion
Training and test data for the application is located in `Applications/TransferLearning/Draw_Classification/res`.
```

**Review by zhoonit — APPROVED** [review]
Minor comments. LGTM

File: Applications/TransferLearning/Draw_Classification/README.md (hunk 3 above)
Comment (kparichay) [reply Applications/TransferLearning/Draw_Classification/README.md]: done.

File: Applications/TransferLearning/Draw_Classification/README.md (hunk 2 above)
Comment (kparichay) [reply Applications/TransferLearning/Draw_Classification/README.md]: done.

File: Applications/TransferLearning/Draw_Classification/README.md (hunk 1 above)
Comment (kparichay) [reply Applications/TransferLearning/Draw_Classification/README.md]: done.

File: Applications/TransferLearning/Draw_Classification/README.md [hunk 4]
```diff
@@ -55,40 +69,31 @@ Input_Shape = 1:1:128         # Input shape (without batch size)
...
-<p align = "center">
-<img src="https://github.com/nnstreamer/nntrainer/blob/master/docs/images/d42b1300-f0cf-11e9-9b6f-6db30def4684.png" width="500" height="300">
-</p>
-
-and the test results for the 8 test cases are below. Step function is used to make more clear.
-As you can see, the test result is ok.
+The training reduces the training loss starting from `1.08` to `0.0048`.
+The test results for the 8 test cases are below. Top-1 prediction is used to check the results.
 
 <p align ="center">
```
Comment (jijoongmoon) [inline Applications/TransferLearning/Draw_Classification/README.md:98]: Let's merge this for now and make another pr for the update which includes nnstreamer for ubuntu and result.

**Review by jijoongmoon — APPROVED** [review]
LGTM
